lib/src/devtools_integration/primitives.dart

```diff
@@ -0,0 +1,10 @@
+
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Name of extension to integrate the leak tracker with DevDools.
+const String memoryLeakTrackingExtensionName = 'ext.dart.memoryLeakTracking';
+
+/// Version of protocol, executed by the application.
+const String appLeakTrackerProtocolVersion = '1';
```

Comment:
Will this be important for caching? 
It may be worth using semver in case we ever need to release incremental patches/fixes etc.

Reply:
I expect it to be simple. For now I want to check version. When we need to introduce breaking changes, we will decide how complicated it should be.